### PR TITLE
mvn: add pt-BR translation

### PR DESCRIPTION
**Pull request by andrik**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt-BR/common/maven.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# mvn
```
Comment (waldyrious) [inline pages.pt-BR/common/maven.md]: O nome do comando não devia ser igual ao nome do ficheiro?

File: pages.pt-BR/common/maven.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Executar as fases `clean` e `package` em um projeto:
+
+`mvn clean package`
+
+- Executar as fases `clean` e `package` em um projeto utilizando um perfil:
+
+`mvn clean -P{{perfil}} package`
+
+- Executar uma classe que possua o método main:
```
Comment (waldyrious) [inline pages.pt-BR/common/maven.md]: Please wrap `main` in backticks.

File: pages.pt-BR/common/maven.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# mvn
+
+> Ferramenta para a criação e gerenciamento de projetos Java.
+> Página Oficial: <https://maven.apache.org/>.
```
Comment (schneiderl) [inline pages.pt-BR/common/maven.md]: Até agora nós usavamos "Homepage" antes dos links. Não devemos continuar assim até que a discussão em #3041 seja terminada?

Comment (mebeim) [reply pages.pt-BR/common/maven.md]: @schneiderl the discussion at #3041 is not about which term to use to indicate links, it's about whether or not core utils should have links to web-hosted man pages. This is a different matter.

Comment (waldyrious) [reply pages.pt-BR/common/maven.md]: @mebeim is right, but indeed the discussion on what the label (when it's present) should be is still ongoing, as I noted [here](https://github.com/tldr-pages/tldr/pull/3030#issuecomment-495006814). That said, this PR shouldn't be held back while we make that decision.

**Pull request merged**